Makefile

```diff
@@ -158,23 +158,25 @@ endif
 	kubectl delete -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
 
 $(eval TMP_ODIR := $(shell mktemp -d))
-gen-manifests: manifests kustomize ## Generates manifests for deploying the operator into flotta-operator.yaml
+gen-manifests: manifests kustomize ## Generates manifests for deploying the operator into $(TARGET)-flotta-operator.yaml
 	@cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 ifeq ($(TARGET), k8s)
 	@sed -i 's/REPLACE_HOSTNAME/$(HOST)/' ./config/k8s/network/ingress.yaml
-	$(KUSTOMIZE) build config/k8s > $(TMP_ODIR)/flotta-operator.yaml
+	$(KUSTOMIZE) build config/k8s > $(TMP_ODIR)/$(TARGET)-flotta-operator.yaml
 	@sed -i 's/$(HOST)/REPLACE_HOSTNAME/' ./config/k8s/network/ingress.yaml
 else ifeq ($(TARGET), ocp)
-	$(KUSTOMIZE) build config/ocp > $(TMP_ODIR)/flotta-operator.yaml
+	$(KUSTOMIZE) build config/ocp > $(TMP_ODIR)/$(TARGET)-flotta-operator.yaml
 else ifeq ($(TARGET), kind)
-	$(KUSTOMIZE) build config/kind > $(TMP_ODIR)/flotta-operator.yaml
+	$(KUSTOMIZE) build config/kind > $(TMP_ODIR)/$(TARGET)-flotta-operator.yaml
 endif
 
 	@cd config/manager && $(KUSTOMIZE) edit set image controller=quay.io/jdzon/flotta-operator:latest
-	@echo -e "\033[92mDeployment file: $(TMP_ODIR)/flotta-operator.yaml\033[0m"
+	@echo -e "\033[92mDeployment file: $(TMP_ODIR)/$(TARGET)-flotta-operator.yaml\033[0m"
 
-release: gen-manifests
-	gh release create v$(VERSION) --notes "Release v$(VERSION) of Flotta Operator" --title "Release v$(VERSION)" '$(TMP_ODIR)/flotta-operator.yaml# Flotta Operator'
+release:
+	TARGET=ocp gen-manifests
+	TARGET=k8s gen-manifests
+	gh release create v$(VERSION) --notes "Release v$(VERSION) of Flotta Operator" --title "Release v$(VERSION)" '$(TMP_ODIR)/ocp-flotta-operator.yaml# OCP Flotta Operator' '$(TMP_ODIR)/k8s-flotta-operator.yaml# K8s Flotta Operator'
```

Comment:
pls change the description from `OCP Flotta Operator` to `Flotta Operator for OCP` and `K8s Flotta Operator` to `Flotta Operator for kubernetes`

Reply:
Done